client/src/Layout/DashboardLayout.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { Box, AppBar, Toolbar, Button, Link, Avatar, Badge, Menu, MenuItem, IconButton } from '@material-ui/core';
+import useStyles from './useStyles';
+import Logo from '../assets/img/logo.png';
+import { useAuth } from '../context/useAuthContext';
+
+interface Props {
+  component: React.ReactNode;
+}
+
+function DashboardLayout({ component }: Props): JSX.Element {
```

Comment:
Please rebase or adjust the base of the PR to only reflect changes pertaining to this feature